apps/total-typescript/src/pages/products/index.tsx

```diff
@@ -172,7 +172,10 @@ const ProductTeaser: React.FC<ProductTeaserProps> = ({
     }
   }, [isPPPAvailable, pppCoupon, isPPPEnabled, setMerchantCoupon])
 
-  if (product.state === 'unavailable' && !purchased) {
+  if (
+    (product.state === 'unavailable' || product.state === 'draft') &&
+    !purchased
```

Comment:
I think the logic behind `!purchased` is to display unavailable products that were purchased. For example, we retired the legacy core volume and the React workshop in TT, but it will still show for those who bought it.

Reply:
makes sense!